FEXCore/Source/Interface/Core/ArchHelpers/Arm64Emitter.cpp

```diff
@@ -335,8 +335,8 @@ namespace x32 {
 }
 
 // We want vixl to not allocate a default buffer. Jit and dispatcher will manually create one.
-Arm64Emitter::Arm64Emitter(FEXCore::Context::ContextImpl *ctx, size_t size)
-  : Emitter(size ? (uint8_t*)FEXCore::Allocator::VirtualAlloc(size, true) : nullptr, size)
+Arm64Emitter::Arm64Emitter(FEXCore::Context::ContextImpl *ctx, void* EmissionPtr, size_t size)
+  : Emitter(reinterpret_cast<uint8_t*>(EmissionPtr), size)
```

Comment:
```suggestion
  : Emitter(static_cast<uint8_t*>(EmissionPtr), size)
```

since this is from void*